src/AutoRest.CSharp/Common/Generation/Writers/RequestWriterHelpers.cs

```diff
@@ -209,7 +199,7 @@ public static void WriteRequestCreation(CodeWriter writer, RestClientMethod clie
                         throw new NotImplementedException(clientMethod.Request.Body?.GetType().FullName);
                 }
 
-                writer.Line($"return {message};");
+                writer.Line($"return {(lowLevel ? request : message)};");
```

Comment:
I do not understand if I needed the `:I` format specifier here or not.  We used it before for the `request` case but not the `message` case and I don't understand enough about the machinery behind all this stuff to know when to use one vs the other.

Reply:
Unsure - @pakrym ?

Reply:
We use `:I` when outputting a string that's an identifier, the code writer would prefix keywords with `@` in that case but it shouldn't be required for `CodeWriterDeclaration` so it's OK to remove.